tests/integration/select/button_focus_core.js

```diff
@@ -15,7 +15,7 @@ function defineTest( testName, clickAction, expectChange ) {
 				}
 			},
 			function( result ) {
-				var activePage = $.mobile.pageContainer.pagecontainer( "getActivePage" );
+				var activePage = $( ".ui-pagecontainer" ).pagecontainer( "getActivePage" );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?